_transactions.md

```diff
@@ -1080,3 +1083,53 @@ Besides the limits listed there, it's also recommended to preemptively [check a
   We currently do not expose an endpoint that lists all the limits that would apply for a given transaction.
   Your application should be prepared to handle these limit errors and forward the relevant information to the user so they can adjust the transaction properties accordingly.
 </aside>
+
+## Transacting with credit/debit cards
+
+The card details provided in this portion of the documentation are not real cards but specifically designed for Sandbox transactions. These can be used to force specific outcomes and test the various flows pertaining to using card data to perform transactions.
```

Comment:
IMHO this makes more sense in the accounts section.

Reply:
I would ask that you please reach out to Ayush to discuss that, as he is the person responsible for deciding that.

Reply:
Given the latest changes we opted to in fact incorporate that suggestion, as it stands there would be no added value in having them separate.